scripts/test:build.sh

```diff
@@ -4,5 +4,4 @@
 # to ensure we will be able to release new versions
 
 (cd packages/react-instantsearch && yarn build) &&
```

Comment:
stray &&

Reply:
I think `&&` is required otherwise the script will continue even if the previous command has failed. Not sure.

Reply:
that's what `set -e` is for. `&&` is implied on every newline then IIRC

Reply:
Ok, I will add it on this file and also in the `boot.sh` then.